features/networking/pod.feature

```diff
@@ -525,3 +525,80 @@ Feature: Pod related networking scenarios
       | ls /var/lib/cni/networks/openshift-sdn/ |
     Then the step should succeed
     And the output should not contain "10.132.2.200"
+
+  # @author zzhao@redhat.com
+  # @case_id OCP-41666
+  # @bug_id 1924741
+  @admin
+  @destructive
+  Scenario: Pod stuck in container creating - failed to run CNI IPAM ADD: failed to allocate for range 0: no IP addresses available in range set
+    Given I switch to cluster admin pseudo user
+    When I run the :label admin command with:
+      | resource  | machineconfigpool         |
+      | name      | worker                    |
+      | key_val   | custom-kubelet=large-pods |
+      | overwrite | true                      |
+    Then the step should succeed
+    Given I obtain test data file "networking/custom_kubelet.yaml"
+    When I run the :create client command with:
+      | f | custom_kubelet.yaml |
+    Then the step should succeed
+    And I wait up to 30 seconds for the steps to pass:
+    """
+    When I run the :get client command with:
+      | resource      | machineconfigpool |
+      | resource_name | worker            |
+    Then the output should match:
+      | .*False\\s+True\\s+False |
+    """
+    And I wait up to 1980 seconds for the steps to pass:
+    """
+    When I run the :get client command with:
+      | resource      | machineconfigpool |
+      | resource_name | worker            |
+    Then the output should match:
+      | .*True\\s+False\\s+False |
+    """
+    Given admin ensures "set-max-pods" kubelet_config is deleted after scenario
+    Given I store the schedulable workers in the :nodes clipboard
+    Given I switch to the first user
+    Given I have a project
+    And evaluation of `project.name` is stored in the :proj1 clipboard
+    Given I obtain test data file "networking/max-pods-without-consume-memory.yaml"
+    When I run oc create over "max-pods-without-consume-memory.yaml" replacing paths:
+      | ["items"][0]["spec"]["template"]["spec"]["nodeName"] | <%= cb.nodes[0].name %> |
+    Then the step should succeed
+    
```

Comment:
@zhaozhanqi How the steps will make sure all 500 ips will be assigned to cb.nodes[0] only not to other nodes in the cluster?

Reply:
yes,  `nodeName` parameter can make pods only be scheduled on this node.